### ISSUE TITLE
Adjust tile rank for transit stations per railway type

### ISSUE THREAD

**Issue by nvkelso** [mapzen/vector-datasource#506]
In London the `kind_tile_rank` sometimes biases towards lower quality service features like light rail & trams to the detriment of larger stations serving "heavy" rail. The same thing is sometimes true in New York (comparing Manhattan to New Jersey when suddenly many light rail appear when only heavy rail appear in Manhattan). There are probably two factors at work here: tile versus meta tile, and railway type. This issue is focused only on railway type.

Since OSM doesn't have trip frequency level data, the route relations act as a good proxy, but the fact that something has more physical infrastructure investment should also act as a signal that there are probably more trips passing thru a station.

On railway type, from @zerebubuth:

```
If we want to count subway routes separately from overground routes, 
then we can do that. Many (although not all) route relations are tagged 
to say whether they're surface, subway, light rail or tram lines. It just 
seems like this makes comparing stations harder (i.e: which is more 
important, the one with 4 surface lines and 2 subway lines, or the one 
with 2 surface lines, 1 light rail line and 3 tram lines?).
```

Proposed solution is to determine the "preferred" / "best" / "dominant" transit type for a station, and to also sort based on that as a tie breaker, maybe weight it, add zooms to the mix?

* heavy rail station A with 2 lines, light rail station with 2 lines.

not

* light rail station A with 2 lines, heavy rail station with 2 lines.

London
12/51.5093/-0.0760

<img width="1025" alt="screen shot 2016-01-21 at 16 12 18" src="https://cloud.githubusercontent.com/assets/853051/12498766/ce08f7b2-c059-11e5-8743-5acdd1060bd6.png">

**Comment by zerebubuth** [comment]
One possible way to do it, which feels like it might be quite intuitive, is to take:

1. The number of mainline rail routes, up to a maximum of 9 and multiply by 100.
2. The number of subway plus light rail routes, up to a total maximum of 9 and multiply by 10.
3. The number of tram / other routes, up to a maximum of 9.

Sum these, and we'll get a score for each station between 0 and 999 (it would be interesting to see if any station in the world gets 999). That can then be the value used for sorting to produce the final rank.

Alternatively, if we think that something with 9 light rail routes should be more important that a single mainline route, then we could add them together after scaling, say `6 * mainline + 2 * light + others` or something to that effect.

Finally, a few observations about London:

1. What's that "tear" near West Silvertown near the river to the east?
2. Almost no stations are showing in the centre of London - are they being suppressed by the road labels? Even if the correct stations aren't showing, I'd expect _some_ stations to show.
3. I forgot that in production we use the "flat nodes" functionality of `osm2pgsql`, so raw nodes aren't stored in the database (on my local machine, I don't use that function), so I'll need to patch the function to try joining on the `planet_osm_point` table instead. Tracking that in #507.

**Comment by nvkelso** [comment]
I dig the proposed solution, let's take it for a spin.

Digging into your comments about London:

1. The "tear" is, I think, an areal way with some Tangram rendering artifacts. Brett is already on it (we saw similar in Hong Kong a couple weeks ago).
2. Road labels don't interact with station labels right now (station labels are "on top of" road labels). It looks like it's being compounded by a filter that worked in the USA but not in London: if the station doesn't have an "area" then it's considered less important. Maybe that should be a contributing factor server side influencing the `kind_tile_rank`, but it doesn't seem useful by itself client side?

Some images (below), with `kind_tile_rank` labeled and stations below the threshold shown tiny icons (and the area filter removed, thus restoring Waterloo and a couple more stations).

Right now subway neighbouring stations that share a couple lines for a small run pop out and get a better rank. True both for subway and light rail – and how there are several light rail routes so they're blowing out nearby subway stations with just 1 or 2 routes.

<img width="982" alt="screen shot 2016-01-22 at 09 42 45" src="https://cloud.githubusercontent.com/assets/853051/12518353/fa79def4-c0ec-11e5-905f-e79c93216320.png">

<img width="1058" alt="screen shot 2016-01-22 at 09 42 36" src="https://cloud.githubusercontent.com/assets/853051/12518357/fab6f4f6-c0ec-11e5-9662-715bc6bc1ff4.png">

<img width="982" alt="screen shot 2016-01-22 at 09 42 28" src="https://cloud.githubusercontent.com/assets/853051/12518355/fab2905a-c0ec-11e5-9bd2-0707170588e9.png">

<img width="998" alt="screen shot 2016-01-22 at 09 42 05" src="https://cloud.githubusercontent.com/assets/853051/12518356/fab57946-c0ec-11e5-8a8d-7e63969360fb.png">

**Comment by zerebubuth** [comment]
I think the fix in #508 will have a big effect here.

Re: areas, I think that's a good tie-break rule, and it'll be more effective once #348 is done. For the moment, it might do more harm than good, as there's a lot of major stations mapped as nodes - Penn Station in NYC, for example.

**Comment by nvkelso** [comment]
Looks like things are better (thanks to #508), but big rail stations are still missing / too low of a rank. Let's pursue the options @zerebubuth describes above for this milestone.

Testing scene file: [tangram-skin-and-bones.zip](https://github.com/mapzen/vector-datasource/files/164169/tangram-skin-and-bones.zip)

The same screenshot areas as above:

<img width="1014" alt="screen shot 2016-03-08 at 15 35 43" src="https://cloud.githubusercontent.com/assets/853051/13620347/ce21b1c6-e543-11e5-80df-69558d4e36cc.png">
<img width="1070" alt="screen shot 2016-03-08 at 15 35 02" src="https://cloud.githubusercontent.com/assets/853051/13620350/ce36ab4e-e543-11e5-9bd1-2e16552c8da7.png">
<img width="1033" alt="screen shot 2016-03-08 at 15 34 47" src="https://cloud.githubusercontent.com/assets/853051/13620349/ce33610a-e543-11e5-9712-7be853da6ca4.png">
<img width="977" alt="screen shot 2016-03-08 at 15 34 27" src="https://cloud.githubusercontent.com/assets/853051/13620348/ce32dbb8-e543-11e5-8a11-dcdc15d93c24.png">

**Comment by nvkelso** [comment]
We had been talking in another issue about setting the `kind` to be more specific (subway, light_rail, etc not just station). Seems like the lookup process would be the same, should we store a new "service" property on the feature as part of the tile_kind_rank calculation?

**Comment by zerebubuth** [comment]
A couple of points here:

London isn't a great place to be testing public transit stuff in OSM. When I was exploring the data, it wasn't one of the cities with the most well-mapped public transit relations. Paris or Berlin would be better, and perhaps we can set aside some time to fix the data in London / NYC?

Public transit relations often span the overground and underground (subway) stations, as well as having light rail and/or international services. This means that a single `kind` or `service` tag doesn't fully represent the services at a station. A couple of possible solutions to that:
* a `service` hierarchy (which would limit styling options in the case that the user prioritises subway / light rail over international services)
* or a `services=[...]` list (which isn't supported by all formats)

**Comment by zerebubuth** [comment]
A third option would be to add boolean flags, e.g: `is_light_rail`, `is_subway`, `is_international`. The downside of that is that it's quite verbose, but it _is_ supported by all formats and railway stations are fairly rare features.

**Comment by zerebubuth** [comment]
Here are the top 20 stations (or halts, tram stops, aerialway stations) in London according to the scoring system set out in [this comment](https://github.com/mapzen/vector-datasource/issues/506#issuecomment-173911390):

```
933	n3662847634	station	"London St. Pancras"
933	n3637436408	station	"London Liverpool Street"
933	n119274464	station	"King's Cross St. Pancras"
631	n441947584	station	"Victoria"
631	n3768010552	station	"London Victoria"
610	w140162700	station	"Stratford"
610	w140162700	station	"Stratford"
600	w170130337	station	"West Ham"
510	w72570259	station	"Queen's Park"
510	n3663368461	station	"Euston"
510	n3663368460	station	"London Euston"
500	w25025418	station	"Canning Town"
431	w302026559	station	"London Paddington"
431	r204439 	station	"London Paddington"
431	n1264482797	station	"Paddington (District, Circle, and Bakerloo Lines)"
420	w289193423	station	"Whitechapel"
420	n2426825788	station	"Whitechapel"
410	w198353814	station	"Highbury & Islington"
401	n3757075887	station	"Wandsworth Common"
400	w41599363	station	"Canada Water"
```

For comparison, here are the ones that I wrote down before starting that I'd expect to see at the top (but in no particular order). The bold ones are not in the above top 20, and show where they are in the list:

* Paddington
* **Waterloo** (114th / 612)
* Victoria
* **London Bridge** (233rd / 612)
* Liverpool Street
* King's Cross / St. Pancras
* Euston
* **Marylebone** (59th / 612)
* **Farringdon** (560th / 612)

The remaining issues seem to be mostly data:

1. Only one of Waterloo's mainline rail routes is detected, because only one is mapped (the yellow highlight on [ÖPNVKarte](http://www.xn--pnvkarte-m4a.de/?zoom=16&lat=51.5006&lon=-0.1142&layers=TBTTT)).
2. None of London Bridge's mainline routes are detected, for a [similar reason](http://www.xn--pnvkarte-m4a.de/?zoom=16&lat=51.50271&lon=-0.08334&layers=TBTTT).
3. Marylebone detects a couple of mainline routes and a subway route, and comes in at 59th - probably just needs more routes mapping, as I'm pretty sure it serves more than 2 routes.
4. Farringdon gets a score of zero because the transport relations are broken and there's no way to "get to" the routes from the station node.

I've played around with the scoring function a little; my intuition is that an interchange station is more important than something which either only serves rail routes or subway routes. But it seems like it's hard to quantify.

**Comment by zerebubuth** [comment]
I should add: there are duplicates in that list, but we'll try to get rid of those "in post" using the `site` or `stop_area` relations which connect them.

**Comment by nvkelso** [comment]
Yes! I was thinking the same thing in https://github.com/mapzen/vector-datasource/issues/587 regarding interchange stations (and even terminal stations). Double their score?

```
If one of these is a "transfer" station where two lines come intersect (but not in a sequence 
running parallel), I could see zoom 12. And if it's the terminal station on the line then zoom 11. 
```

**Comment by zerebubuth** [comment]
Here's the top 20 with the first two digits doubled if they're both non-zero:

```
963     n3662847634     station "London St. Pancras"
963     n3637436408     station "London Liverpool Street"
963     n119274464      station "King's Cross St. Pancras"
961     n441947584      station "Victoria"
961     n3768010552     station "London Victoria"
920     w72570259       station "Queen's Park"
920     w140162700      station "Stratford"
920     w140162700      station "Stratford"
920     n3663368461     station "Euston"
920     n3663368460     station "London Euston"
861     w302026559      station "London Paddington"
861     r204439  station "London Paddington"
861     n1264482797     station "Paddington (District, Circle, and Bakerloo Lines)"
840     w289193423      station "Whitechapel"
840     n2426825788     station "Whitechapel"
820     w198353814      station "Highbury & Islington"
623     n307542330      tram_stop       "Wimbledon"
623     n18089211       station "Wimbledon"
621     r3791635        station "Willesden Junction"
620     n18036791       station "Poplar"
```

It's very similar to the last one, except we've lost West Ham, Canning Town, Wandsworth Common and Canada Water and gained Wimbledon (x2), Willesden Junction and Poplar. Which, on the whole, seems like an improvement.

I'll start porting this to SQL so we can try it out on dev.

**Comment by nvkelso** [comment]
Related: add `osm_site_relation`: https://github.com/mapzen/vector-datasource/issues/590

**Comment by nvkelso** [comment]
Related: #587.

**Comment by zerebubuth** [comment]
Progress dump.

Note: For the screenshots below, I had to turn off all filtering of stations by area, and drop the `min_zoom` for all station queries to 10 (see 1a2f07a7233b67969f12f0ce0d00ce0fdc61e709). This might not be such a bad thing, as stations are relatively rare and we can drop them based on `kind_tile_rank` before the client ever sees them at lower zooms. Alternatively, we can base the `min_zoom` calculation on the transit score instead.

## London, z11

![506-z11-london-1](https://cloud.githubusercontent.com/assets/271360/13755285/6acfd228-ea12-11e5-958a-73df0486db52.jpg)

## London, z12

![506-z12-london-1](https://cloud.githubusercontent.com/assets/271360/13755288/6e0150f2-ea12-11e5-8036-941e5f252b05.jpg)

## Top 20 data:

### London

```
 typ  | osm_id   | score | root_relation_id |           name           
------+------------+-------+------------------+--------------------------
 node |  441947584 |   961 |          1572442 | Victoria
 node | 3768010552 |   961 |          1572442 | London Victoria
 way  |   72570259 |   920 |          3803730 | Queen's Park
 node | 3663368460 |   920 |          1569858 | London Euston
 node |  963506307 |   920 |           199275 | Wembley Central
 way  |  140162700 |   920 |           297366 | Stratford
 node |   10287073 |   920 |           199275 | Wembley Central
 node | 3663368461 |   920 |          1569858 | Euston
 way  |  289193423 |   840 |          3155934 | Whitechapel
 node | 2426825788 |   840 |          3155934 | Whitechapel
 way  |  198353814 |   820 |           206255 | Highbury & Islington
 node | 3662847634 |   663 |          1569873 | London St. Pancras
 node |  119274464 |   663 |          1569873 | King's Cross St. Pancras
 way  |    4959629 |   663 |          1571791 | Liverpool Street
 node | 3637436408 |   663 |          1571791 | London Liverpool Street
 way  |   -3791635 |   621 |          3791636 | Willesden Junction
 way  |   -3791635 |   621 |          3791636 | Willesden Junction
 node |   18036779 |   620 |                  | Westferry
 node |   18036791 |   620 |                  | Poplar
 node |   18020039 |   620 |                  | Limehouse
```

A couple of odd things here:

1. The high rank of Queen's Park is because it has a part-time stop on a national rail route. As far as I can tell, this "part-time" is basically never, so this might be considered a data issue. Wembley is similar: the station node is attached to the railway, along which these national rail routes run, but they don't actually stop at Wembley. One fix for this might be stricter about the roles via which routes are discovered, but this would have unwanted side-effects on other stations.
2. The high rank of the DLR stations (Westferry, Poplar, Limehouse) is because 3 of the DLR routes were tagged as being rail routes. I've since fixed the data to make them `light_rail` routes instead.

### NYC

```
 typ  |   osm_id   | score | root_relation_id |                       name                        
------+------------+-------+------------------+---------------------------------------------------
 node |  895371274 |   902 |                  | New York Penn Station
 way  |  265947358 |   302 |          4637816 | Grand Central Terminal
 node | 2024882763 |   260 |                  | 14th Street (1,2,3)
 node |  597928313 |    70 |                  | 59th Street - Columbus Circle (A,B,C,D,1)
 node |  544356374 |    70 |                  | 34th Street-Herald Square (B,D,F,M,N,Q,R)
 node | 2024979388 |    60 |          2917356 | 59th Street (4,5,6)
 node |  597928309 |    60 |                  | West 4th Street-Washington Square (A,B,C,D,E,F,M)
 node |  597928318 |    60 |          2917356 | 59th Street-Lexington Avenue (N,Q,R)
 node |  591998010 |    50 |          2917492 | 53rd Street-Lexington Avenue (E,M)
 node |  597928319 |    50 |          2917492 | 51st Street (6)
 node |  597928317 |    50 |                  | 23rd Street (F,M,PATH)
 node |  591997657 |    50 |                  | 42nd Street-Grand Central (S,4,5,6,7)
 node |  597928308 |    50 |          3420745 | 14th Street (F,M,PATH)
 node |  597928312 |    40 |                  | Delancey Street-Essex Street (F,J,M,Z)
 node |  597928316 |    40 |                  | 42nd Street - Bryant Park (B,D,F,M)
 node |  597928314 |    40 |                  | 47th-50th Streets - Rockefeller Center (B,D,F,M)
 node | 1692394907 |    40 |                  | Fulton Street-Broadway Nassau (A,C,J,Z)
 node | 2013903253 |    40 |          3420790 | 14th Street-8th Avenue (A,C,E,L)
 node |  591995518 |    40 |                  | Broadway-Lafayette Street (B,D,F,M)
 node | 2052618474 |    40 |          3420670 | 14th Street-Union Square (4,5,6)
```

NYC doesn't have many station relations, so it looks like it's mostly sorting the mainline stations first. The only interchange station is 14th Street, because of [this suspect-looking relation](http://www.openstreetmap.org/relation/3019793). @rmarianski - are there any non-subway routes running through 14th Street?

### San Francisco

```
 typ  |   osm_id   | score | root_relation_id |                  name                   
------+------------+-------+------------------+-----------------------------------------
 way  |   28295229 |   200 |                  | San Francisco 4th & King Street Station
 node | 2160213364 |   100 |                  | 22nd Street
 node | 2160213345 |   100 |                  | Bayshore
 node | 1723738813 |    60 |                  | Embarcadero
 node |  297863037 |    60 |                  | Van Ness Muni
 node |  313885839 |    40 |                  | West Portal Muni
 node |  297863017 |    40 |          1069465 | Castro Muni
 node |  301506011 |    40 |          1069835 | Church St Muni
 node | 1564479547 |    40 |                  | West Portal Muni
 way  |  159782838 |     0 |                  | Balboa Park
 way  |  101546361 |     0 |                  | Daly City
 way  |  136822046 |     0 |                  | Forest Hill
 node | 2150077208 |     0 |                  | 24th Street Mission
 way  |  132246043 |     0 |                  | Glen Park
 node | 2150077196 |     0 |                  | Civic Center / UN Plaza
 node |  301506348 |     0 |                  | Forest Hill Muni
 node | 2150077204 |     0 |                  | Powell Street
 node | 2150077198 |     0 |                  | Embarcadero
 node | 2150077201 |     0 |                  | Montgomery Street
 node | 2150077206 |     0 |                  | 16th Street Mission
```

There's a lot of opportunity for improvement in San Francisco!

**Comment by nvkelso** [comment]
Starting at zoom `10` makes sense to me, and ties in well with other changes in this release related to subway line zoom ranges.

Why aren't there more subway stations visible in central London at zoom 11 and zoom 12? For instance, many of the stations in this screenshot off `prod` seem to be missing (like Barbican)?

<img width="843" alt="screen shot 2016-03-14 at 12 28 20" src="https://cloud.githubusercontent.com/assets/853051/13757143/5293d2a4-e9e0-11e5-8795-c2e2d7382956.png">

I think the 14th St station in NYC correctly includes the PATH commuter rail / other subway like line's [14th St station](https://en.wikipedia.org/wiki/14th_Street_(PATH_station))?

**Comment by nvkelso** [comment]
@zerebubuth should we include `root_relation_id=*` instead of `osm_relation:yes`?

**Comment by zerebubuth** [comment]
>  For instance, many of the stations in this screenshot off prod seem to be missing (like Barbican)?

I can see Barbican - it's about 2/3rds of the way from the left of the image and halfway down it, just above the London Wall neighbourhood label and to the left of the Barbican neighbourhood label.

Why are we showing Aldwych? (It's the one on the dark blue Piccadilly line spur from Holborn) Both the line and the station have been closed for more than 20 years.

**Comment by zerebubuth** [comment]
> Why are we showing Aldwych? (It's the one on the dark blue Piccadilly line spur from Holborn) Both the line and the station have been closed for more than 20 years.

Looks like theres a couple of things going on here...

[Aldwych station](http://www.openstreetmap.org/way/140088942) is being shown because it has `disused:railway=station`, which doesn't match any known POI type, so it's being put in the buildings layer. Because it also has `building=station`, then the label placement for it has `kind:station`. This means we probably should either:

1. Not be displaying POIs on the basis of what's in the buildings layer - that's what the POIs layer is for, and/or
2. Resetting tags on `disused:*` elements.

I would prefer the former, as I'm pretty sure that the latter will result in some strange and difficult-to-track-down bug when there's a `disused:power=plant` which becomes a `tourism=gallery` [for example](https://en.wikipedia.org/wiki/Tate_Modern), and we end up suppressing it.

The reason that the spur is appearing on the map is because the [disused railway](http://www.openstreetmap.org/way/139415643) is a member of the [route relation](http://www.openstreetmap.org/relation/102788#map=14/51.5116/-0.1192). There's a good case why it shouldn't be, but it _is_ tagged `railway=disused`, so we would be able to filter it out with some changes to `osm2pgsql`.

**Comment by nvkelso** [comment]
Yeah, I see `Barbican` on my colorful screenshot, but not in your London screenshots from zoom 11 and 12?

Good news, looks like `Aldwych (Closed)` station will be dropped once https://github.com/mapzen/vector-datasource/issues/483 lands:

<img width="563" alt="screen shot 2016-03-14 at 13 52 43" src="https://cloud.githubusercontent.com/assets/853051/13759585/21c382ee-e9ec-11e5-8404-5f884f559ff9.png">

**Comment by zerebubuth** [comment]
Oh, Barbican is there - I just stopped the style from displaying anything with `kind_tile_rank>16` because it was getting too cluttered. At z12, Barbican is in tile 12/2046/1361, which covers a huge chunk of central  London, so a rank of 20 doesn't seem unreasonable:

![tile 12/2046/1361](http://c.tile.openstreetmap.org/12/2046/1361.png)

**Comment by zerebubuth** [comment]
> Good news, looks like Aldwych (Closed) station will be dropped once #483 lands

Hmm... how about [Great Falls Depot](http://www.openstreetmap.org/way/326483161) or the old [King's Cross Pentonville Road Entrance](http://www.openstreetmap.org/way/137825940)?

**Comment by nvkelso** [comment]
> Oh, Barbican is there - I just stopped the style from displaying anything with kind_tile_rank>16 because it was getting too cluttered.

Cool, looking forward to seeing this on `dev`.

> 1. Not be displaying POIs on the basis of what's in the buildings layer - that's what the POIs layer is for, and/or

Aha, good catch! Looks like that's a long standing bug in the scene files where we were displaying icons for building label placement as if they were POIs (and we shouldn't):

* https://github.com/nvkelso/tangram-skin-and-bones/issues/24
* https://github.com/tangrams/eraser-map/issues/168

Here's a preview:

<img width="842" alt="screen shot 2016-03-14 at 14 31 07" src="https://cloud.githubusercontent.com/assets/853051/13760657/7886f1e2-e9f1-11e5-9867-05cf98782626.png">


> Hmm... how about Great Falls Depot or the old King's Cross Pentonville Road Entrance?

Agree, here's the above solution applied to these two buildings:

<img width="619" alt="screen shot 2016-03-14 at 14 32 54" src="https://cloud.githubusercontent.com/assets/853051/13760707/b8573f52-e9f1-11e5-9ebc-6a2c0bd59307.png">
<img width="373" alt="screen shot 2016-03-14 at 14 32 30" src="https://cloud.githubusercontent.com/assets/853051/13760708/b86f232e-e9f1-11e5-91b6-3ba20bc5bf58.png">

**Comment by nvkelso** [comment]
Change is looking fantastic in NYC, London, Paris! As you note, SF is so-so but that's because it needs more linking-up on the data side. There aren't any regressions over v0.8 prod so we're good.

**One question:**

I expected **two Waterloo stations**, one for rail, one for subway in zoom 16 tile, but I don't see the subway station (`3638795618`). Are we too aggressively de-duplicating features? Perhaps they split apart at zoom 17 when originally implemented in the earlier release, but now that we're "meta-tiling" at zoom 16 we need to adjust the `queries.yaml` end_zoom? Starting at zoom 14 I expect to see individual features. Is it as simple as queries.yaml, or does other logic need to change?

* https://www.openstreetmap.org/edit?node=3638795617#map=19/51.50284/-0.11280
* https://www.openstreetmap.org/edit?node=3638795618#map=19/51.50299/-0.11396

<img width="360" alt="screen shot 2016-03-24 at 02 12 43" src="https://cloud.githubusercontent.com/assets/853051/14012893/1bf44f2a-f166-11e5-9300-87d166556182.png">

**Comment by zerebubuth** [comment]
I think zoom 14 is far too early to start showing individual features. At zoom 14, the two Waterloo station points are 14px apart, which is just enough for two confusingly tiny 12px icons to be drawn (but neither labelled). Even if one had a label, it would be confusing as to which point was being labelled.

Zoom 15 is not much better, and I think it's probably best to split the features at z16. I'm not quite sure whether you're saying that's never happening since we stopped generating >z16 tiles or not?

**Comment by nvkelso** [comment]
Sure, let's give zoom 15 try (to stop dedup'ing). There's certainly room to place the multiple icons by then, and it looks weird / broken to not show them when everyone else does (labeling is labeling, shrug). Looks like the same problem is happening at **Euston**?

<img width="187" alt="screen shot 2016-03-24 at 08 18 54" src="https://cloud.githubusercontent.com/assets/853051/14021715/f272d9bc-f199-11e5-9744-f57a933765cd.png">
<img width="550" alt="screen shot 2016-03-24 at 08 16 14" src="https://cloud.githubusercontent.com/assets/853051/14021717/f2b8ceea-f199-11e5-907d-b5e2efa0b183.png">
<img width="311" alt="screen shot 2016-03-24 at 08 16 05" src="https://cloud.githubusercontent.com/assets/853051/14021718/f2b98272-f199-11e5-97b2-3d0156e09328.png">
<img width="568" alt="screen shot 2016-03-24 at 08 14 05" src="https://cloud.githubusercontent.com/assets/853051/14021721/f2e29acc-f199-11e5-867d-37570ae9e963.png">
<img width="368" alt="screen shot 2016-03-24 at 08 13 47" src="https://cloud.githubusercontent.com/assets/853051/14021719/f2b99f00-f199-11e5-9dee-aa280bad04b3.png">
<img width="232" alt="screen shot 2016-03-24 at 08 13 34" src="https://cloud.githubusercontent.com/assets/853051/14021720/f2b9dc22-f199-11e5-9381-f36441e96ad6.png">

**Comment by nvkelso** [comment]
Everything else looks good, and this isn't causing a regression. Let's pick up the zoom 15+ stuff in the next release with https://github.com/mapzen/vector-datasource/issues/637.
